slide-deck.css

```diff
@@ -3,19 +3,59 @@ slide-deck {
   container: slide-deck / inline-size;
   display: grid;
 
-  &[slide-view=grid] {
+  &[slide-view=grid],
+  &[slide-view=script] {
     --slide-ratio: 16/9;
-    --target-outline: medium dotted;
     --target-margin: var(--gap);
-    gap: var(--gap);
-    grid-template-columns: repeat(auto-fill, minmax(min(50ch, 100%), 1fr));
+    --target-outline: medium dotted;
+    --target-outline-offset: calc(var(--gap) * 0.5);
+    gap: var(--row-gap, var(--gap)) var(--column-gap, var(--gap));
     padding: var(--gap);
   }
 
-  &[slide-view=list] {
+  &[slide-view=grid] {
+    --slide-note-margin-block: var(--gap);
+    grid-template-columns: repeat(auto-fill, minmax(min(50ch, 100%), 1fr));
+  }
+
+  &[slide-view=solo] {
     --slide-height: 100svh;
```

Comment:
Maybe this should be set on the grid instead? eg

```css
grid-auto-rows: minmax(100svh, auto);
```

Reply:
@mirisuzanne  when I played with this, the slides with note don't take up the full height then. Am I missing something?

Reply:
Oh, yeah, but I think we should be hiding the notes in that view? And then it will work.

Reply:
@mirisuzanne It looks like we still need the slide height for the container slide-items (or some additional edits to the CSS). The canvas only items are fine with the grid-rows sizing. Just pushed an update that will make it easier to see if you uncheck the --slide-height on [slide-view=solo] [slide-item] 

![Screenshot 2024-01-11 at 11 03 18 AM](https://github.com/oddbird/slide-deck/assets/1581694/b40f640c-baf7-4e0c-9878-962ea7b43c61)


![Screenshot 2024-01-11 at 11 03 22 AM](https://github.com/oddbird/slide-deck/assets/1581694/cd73a543-0467-4bea-899b-db527e3ba7b7)